kratos/python_scripts/KratosUnittest.py

```diff
@@ -76,21 +76,22 @@ def Usage():
         '\t python kratos_run_tests [-l level] [-v verbosity]',
         'Options',
         '\t -h, --help: Shows this command',
-        '\t -l, --level: Minimum level of detail of the tests: \'all\'(Default) \'(nightly)\' \'(small)\'',  # noqa
-        '\t              For MPI tests, use the equivalent distributed test suites: \'(mpi_all)\', \'(mpi_nightly)\' \'(mpi_small)\'',
-        '\t -v, --verbose: Verbosity level: 0, 1 (Default), 2'
+        '\t -l, --level: Minimum level of detail of the tests: \'all\'(Default) \'(nightly)\' \'(small)\' \'(validation)\'',  # noqa
+        '\t -v, --verbose: Verbosity level: 0, 1 (Default), 2',
+        '\t --using-mpi: If running in MPI and executing the MPI-tests'
     ]
-
+    from KratosMultiphysics import Logger
     for l in lines:
-        print(l)
+        Logger.PrintInfo(l) # using the logger to only print once in MPI
 
 
 def runTests(tests):
     verbose_values = [0, 1, 2]
-    level_values = ['all', 'small', 'nightly', 'validation', 'mpi_all', 'mpi_small', 'mpi_nightly', 'mpi_validation']
+    level_values = ['all', 'small', 'nightly', 'validation']
```

Comment:
The original idea was to place the mpi tests on separate queues, what would be your plan in this sense? (or, to put it in a different way, should mpi runs also call the serial tests of the corresponding suite?)

Reply:
I haven't removed them, only changed how they are selected:
=> If I run the tests with `--using-mpi` then automatically the corresponding mpi test queue is selected (see [here](https://github.com/KratosMultiphysics/Kratos/blob/core/mpi-testing-wip/kratos/python_scripts/KratosUnittest.py#L134-L135))
i.e. if you launch with `--using-mpi` then `small` gets converted to `mpi_small`

We can discuss it in person, as I said it has been some time since I did this